client/src/store/reducers/userReducer.js

```diff
@@ -1,3 +1,4 @@
+import { NewReleasesTwoTone } from "@material-ui/icons";
```

Comment:
are we using this icon in our reducer?

Reply:
This import is unused in the user reducer and has been deleted